app/src/main/java/com/example/vybe/AddEditVibeActivity.java

```diff
@@ -304,4 +310,27 @@ public String formatDateTime(LocalDateTime dateTime) {
         return dateTime.format(DateTimeFormatter.ofPattern("MMM dd yyyy, hh:mm a"));
     }
 
+    /**
+     * This will load an image from Firebase Storage into an ImageView
+     * @param imageView
+     *      The destination ImageView
+     * @param imagePath
+     *      Path to the image in Firebase Storage
+     */
+    public void loadImageFirebase(ImageView imageView, String imagePath){
+        StorageReference storageRef = FirebaseStorage.getInstance().getReference();
```

Comment:
are these names okay? might get confusing

Reply:
It should be fine to leave them like that. We can change if it does get confusing